examples/pybullet/turtlebot_rovers/streams.py

```diff
@@ -46,7 +46,7 @@ def get_reachable_test(problem, iterations=10, **kwargs):
     initial_confs = {rover: Conf(rover, get_base_joints(rover))
                      for rover in problem.rovers}
     # TODO: restarts -> max_restarts
-    motion_fn = get_motion_fn(problem, restarts=0, max_iterations=iterations, smooth=0, **kwargs)
+    motion_fn = get_motion_fn(problem, restarts=0, iterations=iterations, smooth=0, **kwargs)
```

Comment:
In the latest commit of the `motion` submodule, this argument name was changed to `max_iterations` ... so I'm not sure what we want to do here, actually. Just pull in a newer SHA of the `motion` submodule?

Reply:
Hmm did you change anything else other than the variable name `max_iterations` to `iternations` to make the turtlebot rover example work? I did the same and still got the same error I reported in the previous comment.

Reply:
Yes! In the `pybullet-utils` repo, namely `utils.py` I also had to make this change in the definition of the `wrapped` function:

```
    #from functools import lru_cache as cache
    # @cache(maxsize=None, typed=False)
    if sys.version_info >= (3, 9):
        from functools import cache # New in version 3.9
        @cache
        def wrapped(*args, **local_kwargs):
            return normal(*args, **local_kwargs)
        return wrapped
    else:
        // This was the old function
        @cache_decorator
        def wrapped(*args, **local_kwargs):
            return normal(*args, **local_kwargs)
        return wrapped 
```